docs/source/model_design/cause.rst

```diff
@@ -666,12 +666,148 @@ Severity Splits
 Mortality Impacts
 +++++++++++++++++
 
+All-Cause Mortality
+^^^^^^^^^^^^^^^^^^^
+
+All-cause mortality rate (ACMR) is a measure of total deaths (due to all 
+causes) per person-year in the overall age-, sex-, time-, and 
+location-specific population. Specifically, 
+
+.. math::
+
+  \frac{\text{number of deaths due to all causes}}{\text{person-years in the overall population}}
+
+For instance, the global ACMR for the early neonatal age group (0-6 days) 
+in 2017 was approximately 70,000 deaths per 100,000 person-years (0.7 
+deaths per person-year). However, the global ACMR for the post neonatal 
+age group (1 month to 1 year) in 2017 was approximately 1,000 deaths per 
+100,000 person-years (0.01 deaths per person-year). By comparing ACMRs 
+between these age groups, we can see that individuals die at a higher rate 
+during the early neonatal period than the post neonatal period. 
+
+Notably, ACMR is used both for validation of Vivarium simulations, as well as 
+for estimating simulation mortality rates (see the `Mortality Hazards`_ 
+section for more detail).
+
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+Cause-specific mortality rate (CSMR) is a measure of deaths due to a 
+particular cause (or group of causes) per person-year in the overall age-, 
+sex-, time-, and location-specific population. Specifically, 
```

Comment:
Embedded in this innocuous statement is the pernicious assumption that every death has precisely one underlying cause.  I propose we call this assumption out around this point in the text, and link to the causality material for readers who want to dive down that rabbit hole.

Reply:
Thanks, Abie! I included some notes about this in the latest update.